plugins/backfill/storage/transformerExporter.go

```diff
@@ -35,28 +14,6 @@ var Exporter exporter
 
 func (e exporter) Export() ([]event.TransformerInitializer, []storage.TransformerInitializer, []interface1.ContractTransformerInitializer) {
 	return []event.TransformerInitializer{},
-		[]storage.TransformerInitializer{
-			flap_v1_0_9.StorageTransformerInitializer,
-			flip_bat_a_v1_0_9.StorageTransformerInitializer,
-			flip_eth_a_v1_0_9.StorageTransformerInitializer,
-			flip_knc_a_v1_0_8.StorageTransformerInitializer,
-			flip_knc_a_v1_0_9.StorageTransformerInitializer,
-			flip_mana_a_v1_0_9.StorageTransformerInitializer,
-			flip_tusd_a_v1_0_7.StorageTransformerInitializer,
-			flip_tusd_a_v1_0_9.StorageTransformerInitializer,
-			flip_usdc_a_v1_0_4.StorageTransformerInitializer,
-			flip_usdc_a_v1_0_9.StorageTransformerInitializer,
-			flip_usdc_b_v1_0_7.StorageTransformerInitializer,
-			flip_usdc_b_v1_0_9.StorageTransformerInitializer,
-			flip_wbtc_a_v1_0_6.StorageTransformerInitializer,
-			flip_wbtc_a_v1_0_9.StorageTransformerInitializer,
-			flip_zrx_a_v1_0_8.StorageTransformerInitializer,
-			flip_zrx_a_v1_0_9.StorageTransformerInitializer,
-			flop_v1_0_9.StorageTransformerInitializer,
-			median_knc.StorageTransformerInitializer,
-			median_mana.StorageTransformerInitializer,
-			median_wbtc.StorageTransformerInitializer,
-			median_zrx.StorageTransformerInitializer,
-		},
+		[]storage.TransformerInitializer{},
```

Comment:
I did - was thinking that it would be easier to do a backfill for all of the new cat transformers in one go, than having to change it for each new transformer.